tests/lib/volumeRendererParameters.cpp

```diff
@@ -44,21 +43,6 @@ BOOST_AUTO_TEST_CASE(defaultValues)
 #endif
 }
 
-BOOST_AUTO_TEST_CASE(genericParams)
-{
-    livre::VolumeRendererParameters params;
-    params.setMinLOD( 2 );
-    const zerobuf::Data& zerobuf = params.toBinary();
-    BOOST_CHECK( zerobuf.size > 0 );
```

Comment:
first part of the test "livre::VolumeRendererParameters params;" can still be useful ?